src/net/netman.lua

```diff
@@ -0,0 +1,81 @@
+local ctx = require("gamectx.global")
+local logger = require("logger.logger")
+
+local netman = {}
+
+local CLIENT_CODE = "net/client.lua"
+local SERVER_CODE = "net/server.lua"
+
+-- types of commands
+netman.CMD_CONNECT = 1
+netman.CMD_JOIN = 2
+netman.CMD_LEAVE = 3
+netman.CMD_WELCOME = 4
+netman.CMD_STOP = 5
+
+-- types of data we can request to send
+netman.SEND_COORD = 1
+
+-- disconnect reasons
+netman.DISCONNECT_BAD_VERSION = 1
+netman.DISCONNECT_LEFT = 2
```

Comment:
Part of me thinks this should be `0`, but only because `DISCONNECT_LEFT` seems like a normal reason to exit, vs `DISCONNECT_BAD_VERSION`, which seems like an error. You may not have been trying to channel Unix exit codes here though, so feel free to disregard.

Reply:
Yeah, that's a good point.  The default rc is 0, and that seems a good value to use for the default reason.